client/src/components/fhirpipe/index.tsx

```diff
@@ -0,0 +1,255 @@
+import axios from 'axios';
+import { Button, Collapse, MenuItem, Switch } from '@blueprintjs/core';
+import { ItemPredicate, MultiSelect, Select } from '@blueprintjs/select';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useApolloClient, useQuery } from '@apollo/react-hooks';
+
+import { loader } from 'graphql.macro';
+
+import Navbar from 'components/navbar';
+import { IReduxStore } from 'types';
+
+import { FHIRPIPE_URI } from '../../constants';
+
+import './style.scss';
+
+interface Source {
+  id: string;
+  name: string;
+  template: {
+    name: string;
+  };
+}
+interface Resource {
+  id: string;
+  label: string;
+  definition: {
+    type: string;
+  };
+}
+
+// GRAPHQL
+const qSources = loader('src/graphql/queries/sources.graphql');
+const qResources = loader('src/graphql/queries/resources.graphql');
+const qCredentialForSource = loader(
+  'src/graphql/queries/credentialIdForSource.graphql'
+);
+
+const FhirpipeView = () => {
+  const client = useApolloClient();
+
+  const toaster = useSelector((state: IReduxStore) => state.toaster);
+
+  const [selectedSource, setSelectedSource] = useState({} as Source);
+  const [selectedResources, setSelectedResources] = useState([] as Resource[]);
+  const [isMenuDisplayed, setMenuDisplayed] = useState(false);
+  const [bypassValidation, setBypassValidation] = useState(false);
+  const [resetStore, setResetStore] = useState(false);
+  const [multiprocessing, setMultiprocessing] = useState(false);
+
+  const { data: dataSources } = useQuery(qSources, {
+    fetchPolicy: 'network-only'
+  });
+  const { data: dataResources } = useQuery(qResources, {
+    fetchPolicy: 'network-only'
```

Comment:
why network only  ?

Reply:
Because created sources and resources are not added to cache when created, I wanted to be sure that we don't need to refresh when landing on this page

Reply:
ok